common/log/log.go

```diff
@@ -85,7 +87,19 @@ func GetGID() uint64 {
 	return n
 }
 
-var Log *Logger
+var globalLogger unsafe.Pointer
+
+func Log() *Logger {
```

Comment:
just a comment.
this function seems not necessary to be public.

Reply:
used in http rpc to set the log level.